simplesat/tests/test_scenarios_policy.py

```diff
@@ -62,6 +63,13 @@ def test_ipython(self):
     def test_iris(self):
         self._check_solution("iris.yaml")
 
+    def test_no_candidate(self):
+        self.assertRaises(
```

Comment:
We don't care about pre `2.6` compat, so we should use the context manager form instead:

```
with self.assertRaises(NoPackageFound):
    self._check_solution(...)
```